gapic/src/main/com/google/gapid/perfetto/models/CounterTrack.java

```diff
@@ -39,17 +40,19 @@
 
 public class CounterTrack extends Track.WithQueryEngine<CounterTrack.Data> {
   private static final String VIEW_SQL_DELTA =
-      "select ts + 1 ts, lead(ts) over win - ts dur, lead(value) over win value " +
+      "select ts + 1 ts, lead(ts) over win - ts dur, lead(value) over win value, id " +
```

Comment:
This should be 'lead(id) over win id' instead of id.

Reply:
Nice catch, thanks! Done.